### ISSUE TITLE
Tests should pass

### ISSUE THREAD

**Issue by jspjutNV** [NVlabs/abstract-fps#145]
The recently added test infrastructure from PR #134 includes 8 tests that are currently failing. These should be fixed.

The failing tests are the following:

- [x]  `FPSciTests.KillTargetFront`
- [x]  `FPSciTests.KillTargetFrontHoldclick`
- [x]  `FPSciTests.KillTargetRightRotate`
- [x]  `FPSciTests.KillTargetRightTranslate`
- [x]  `FPSciTests.ResetCamera`
- [x]  `FPSciTests.RotateCamera`
- [x]  `FPSciTests.MoveCamera`
- [x]  `FPSciTests.TestAutoFire`

**Comment by bboudaoud-nv** [comment]
Several of these tests appeared to be failing due to an interaction with the user window. By adding `showMenuOnStartup = false;` to the test experiment config I was able to get these running successfully. This inclues:

* `FPSciTests.ResetCamera`
* `FPSciTests.RotateCamera`
* `FPSciTests.MoveCamera`

**Comment by bboudaoud-nv** [comment]
I was able to get `KillTargetFront` and `KillTargetForntHoldclick` running by:

1. Updating the in-code target spawn distance to bring them forward (to 0.5m)
2. Creating a new `FPSciTests::injectFire()` routine to handle adding a frame of delay between mouse button down and up events
3. Modifying the experiment config file to increase weapon damage and turn off gravity

In this process I noticed that using `G3D Simple Cornell Box (Empty CO)` is not the best choice because it appears to neither have a floor for the player to stand on, nor have geometry at sufficient distance not to occlude the target. We might want to consider uploading/using something like our `FPSci Square Map` for testing since it meets these requirements/is designed for FPSci.
